tests/test.rs

```diff
@@ -428,7 +428,7 @@ fn test_integration() {
 
             if !cfg!(windows) {
                 println!("GCC");
-                let gcc_version = get_version("gcc");
+                let gcc_version = get_version("g++");
```

Comment:
Why this change?

Reply:
I've several gcc versions on my computer and with this modif it's easier to replace g++ by g++-7 ;)